src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,9 +1,270 @@
 import React, { Component } from 'react';
+import ProductPhoto from './ProductPhoto';
+import ProductSize from './ProductSize';
+import ProductBonus from './ProductBonus';
+import ShoppingBag from './ShoppingBag';
 import './ProductDetail.scss';
 
 class ProductDetail extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      productInfoList: [],
+      imageNum: 0,
+      isSize: true,
+      isBonus: true,
+      isShoppingBag: true,
+      size: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/productData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productInfoList: data.results,
+        });
+      });
+  }
+
+  postProduct = () => {
+    const { productInfoList } = this.state;
+    fetch('http://10.58.7.79:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        product_id: productInfoList.product_id,
+      }),
+    });
+  };
```

Comment:
```suggestion
  postProduct = () => {
    const { productInfoList } = this.state;
    
    fetch(`${API}/users/signup`, {
      method: 'POST',
      body: JSON.stringify({
        product_id: productInfoList.product_id,
      }),
    });
  };
```
- 구분되는 로직 끼리는 한줄 띄워 주세요
- 주소 같은 경우 변경이 잦은데 변경이 있을 때마다 매 호출하는 곳마다 변경해주기 보다는 src/config.js 같은 경로로 분리하여 사용하는게 좋습니다

Reply:
넵 수정하겠습니다 !